src/pages/Manager.vue

```diff
@@ -362,7 +363,11 @@ import GameRunningModal from '../components/modals/GameRunningModal.vue';
 		}
 
 		@Watch('thunderstoreSearchFilter')
-		onThunderstoreFilterUpdate() {
+        onThunderstoreFilterUpdate = debounce(() => {
```

Comment:
Nitpick: could just pass `this.performThunderstoreFilterUpdate` instead of wrapping it into an arrow function.